src/Command/InstallModxCommand.php

```diff
@@ -35,6 +35,12 @@ protected function configure()
                 'The version of MODX to install, in the format 2.3.2-pl. Leave empty or specify "latest" to install the last stable release.',
                 'latest'
             )
+            ->addArgument(
```

Comment:
```suggestion
            ->addOption(
```

For a cli-flag, instead of "addArgument" s-console wants "addOption"

Reply:
here the docs
https://symfony.com/doc/4.1/console/input.html#using-command-options

Reply:
Gosh, the finishing touch has been sitting here all this time.. Thanks @sebastian-marinescu! It's changed now.